src/main/java/org/lightcouch/legacy/v1/Attachment.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.lightcouch;
+package org.lightcouch.legacy.v1;
```

Comment:
nit: how about `com.vi.lightcouch1`?

Reply:
Responded to similar review comment above: https://github.com/VirtualInstruments/LightCouch/pull/1#discussion_r1152887214